dev-qt/qtwebengine/qtwebengine-5.7.0.ebuild

```diff
@@ -95,3 +98,9 @@ src_configure() {
 	)
 	qt5-build_src_configure
 }
+
+src_install() {
+	qt5-build_src_install
+
+	pax-mark m "${ED}"/usr/$(get_libdir)/qt5/libexec/QtWebEngineProcess
```

Comment:
use `"${D%/}${QT5_LIBEXECDIR}"/QtWebEngineProcess`

Also, we pax-mark unconditionally here, but we condition on `use pax_kernel` for the patch above... what's the recommended approach? The patch is needed to pax-mark a binary used during the build process, I see there's also `host-is-pax` in the eclass for this purpose, but I'm a little confused. @blueness can you shed some light please?